action.yml

```
@@ -14,7 +16,19 @@ inputs:
     required: false
     default: 'x64'
   jdkFile:
-    description: 'Path to where the compressed JDK is located. The path could be in your source repository or a local path on the agent.'
+    description: 'Path to where the compressed JDK is located. The path could
+       be in your source repository or a local path on the agent.'
+    required: false
+  server-id:
+    description: 'ID of the distributionManagement repository in the pom.xml
+       file.'
+    required: false
+  username:
+    description: 'Username for authentication to the Apache Maven server.'
+    required: false
+  password:
+    description: 'Passowrd or token for authentication to the Apache Maven
+       server.'
```

```suggestion
       repository.'
```
Per the suggestion of @giltene